### ISSUE TITLE
training details

### ISSUE THREAD

**Issue by jh-yu** [liangfu/mxnet-mobilenet-v2#2]
Hi, thanks for your contribution first. I am quite interested in training details of mobilenet v2. Is the current train_imagenet.py exactly the setting to reproduce 72.45 accuracy? Thanks.

**Comment by liangfu** [comment]
I'm glad you're interested in the project.
Yes, it's 72.45 in the validation set of ImageNet 2012, I have no accuracy results in the test set though.
Following the MobileNetV2 paper, I use initial learning rate of 0.045 and decrease rate of 0.98, after learning rate reach 1e-4, random scaling range is changed from 0.533~1 to 0.533~0.534, and augmentation level is changed from 2 to 1, leaving no augmentation in original image color. In the RecordIO, I use is the recommended size in ResNet paper, which is 480xN for each input image.

**Comment by jh-yu** [comment]
Thanks a lot!
